warehouse/models/mart/transit_database/dim_organizations.sql

```diff
@@ -44,7 +45,7 @@ dim_organizations AS (
     FROM dim
     LEFT JOIN ntd_agency_to_organization ntd_to_org
         ON dim.source_record_id = ntd_to_org.organization_record_id
-    LEFT JOIN dim_annual_database_agency_information AS ntd
+    LEFT JOIN dim_annual_ntd_agency_information AS ntd
         -- TODO: make this historical and use a bridge table
```

Comment:
This `TODO` can be removed. Is there an `_is_current` column in `dim_annual_ntd_agency_information`? Also see above I think we may not even need this second join.

Reply:
Resolved in [84691c6](https://github.com/cal-itp/data-infra/pull/2582/commits/84691c609797dde00a57a56a90e3b0e939a7442d).